libgalois/include/katana/PODVector.h

```diff
@@ -34,6 +36,8 @@
 
 namespace katana {
 
+enum class MemoryPinType : uint8_t { Usual = 0, Pinned = 1 };
```

Comment:
I don't like `Usual`. What about `NotPinned`?

Also let's not limit the size unless we actually expect it to matter. This is a premature optimization that can result in ABI compatibility issues.

Reply:
We can enhance it further and include GPU memory in MemoryTypes and I agree with Arthur that `Usual` should be `NonPinned`

Reply:
How about `Swappable`? I try to avoid negations in the naming and the code, just think of `if(x != MemoryPinType::NotPinned)`. Pinned memory cannot go to a swap file, in contrast to swappable memory.